docs/1.1-DRAFT/workflows.md

```diff
@@ -169,26 +175,70 @@ A workflow diagram may still be provided even if there is no programmatic `Softw
+* [keywords] with comma separated list of keywords that categorise this workflow
+* [maintainer] to indicate a [Person] or [Organization] responsible for maintaining the workflow, specially if different from [creator]
+* [producer] the [Organization] or [Person] that, when different from [creator] and [maintainer], can be considered to _produce_ the workflow; typically a [Project]
+* [publisher] the [Organization] or [Person] that is making the workflow publicly available, e.g. `https://workflowhub.eu/`, `https://usegalaxy.eu/` or `https://usegalaxy.eu/`
+* [runtimePlatform] the runtime platform or script interpreter as an informal text string, e.g. `"Python2.3"`, `"Galaxy"` (see also [programmingLanguage] above)
+* [softwareRequirements] indicating [SoftwareApplication]s or [SoftwareSourceCode]s needed for this workflow, its steps and workflow engine to execute, e.g. `https://bioconda.github.io/`, `https://docs.docker.com/engine/`, `https://bio.tools/samtools`
+* [targetProduct] to indicate a [SoftwareApplication] for the operating system or product which this code applies to, e.g. `https://www.microsoft.com/en-gb/windows/get-windows-10` or `https://releases.ubuntu.com/20.04/`
+
+The [ComputationalWorkflow] data entity MAY also describe, where relevant:
+* [alternateName] to indicate an alternative [name] that help identify the workflow, e.g. `nf-core/chipseq`
+* [conditionsOfAccess] to indicate restrictions on the availability of this workflow, e.g. `"To protect endangered wildlife, this biodiversity workflow is only available to approved research projects"`
+* [dateModified] to indicate a date later than [dateCreated] when this workflow was modified
+* [datePublished] to indicate a date different from [dateCreated] for when this workflow was made public
+* [encodingFormat] to indicate the [IANA media type](data-entities.md#adding-detailed-descriptions-of-encodings), e.g. `"application/xml"` or `"application/json"` (see also [programmingLanguage] above)
+* [image] to indicate an [ImageObject] that represents the workflow, e.g. a workflow diagram
+* [identifier] to provide a unique identifier for this workflow, e.g. `https://doi.org/10.5281/zenodo.3966161` or `urn:uuid:0a98135b-3e48-4f76-aed8-4a7f6421bd74`
+
+### Describing inputs and outputs
+
+The input and output _parameters_ for a workflow or script can be given with `input` and `output` to [FormalParameter]
+contextual entities. Note that this entity usually represent a _potential_ input/output value in a reusable
+workflow, much like [function parameter definitions](https://en.wikipedia.org/wiki/Parameter_(computer_programming))
+in general programming.
+
+When complying with the
+[BioSchemas FormalParameter profile](https://bioschemas.org/profiles/FormalParameter/0.1-DRAFT-2020_06_23/)
+the _contextual entities_ for [FormalParameter], referenced by `input` or `output`, MUST describe:
 
 * [name] given the programmatic name for the parameter binding
```

Comment:
First time the notion of parameter binding is included, but it's not defined. Maybe an example would clarify things here

Reply:
Changed to just "parameter"